huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_read_replica_instance_test.go

```diff
@@ -107,6 +107,13 @@ func TestAccReadReplicaInstance_withEpsId(t *testing.T) {
 		ProviderFactories: acceptance.TestAccProviderFactories,
 		CheckDestroy:      testAccCheckRdsInstanceDestroy(resourceType),
 		Steps: []resource.TestStep{
+			{
+				Config: testAccReadReplicaInstance_baseEpsId(name, dbPwd),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckRdsInstanceExists(resourceName, &replica),
+					resource.TestCheckResourceAttr(resourceName, "enterprise_project_id", "0"),
+				),
+			},
 			{
 				Config: testAccReadReplicaInstance_withEpsId(name, dbPwd),
```

Comment:
how about add a parameter in the function to set the enterprise project ID?

Reply:
done